src/locales/locale-en.json

```diff
@@ -333,8 +333,8 @@
     "NO_DONE_ACTIVITIES_HINT": "You currently have nothing to give feedback to. Why not join an activity?",
     "NUMBER_ACTIVITIES": "{count} activities done",
     "NUMBER_FEEDBACK": "{count} feedback entries given",
-    "SAVED_FOOD": "At least 1 kg was saved here. Thank you! | People gave feedback for {amount} kg of saved food already! Keep up the great work!",
-    "WEIGHT_SUM": "{sum} kg of food saved"
+    "SAVED": "At least 1 kg was saved here. Thank you! | People gave feedback for {amount} kg saved already! Keep up the great work!",
```

Comment:
'People gave feedback for 10 kg saved' - sounds a little strange to me, maybe we can rephrase this somehow?

Reply:
Maybe just __x kg was saved__? Seems to fit the grammer of the existing one: __At least 1 kg was saved here__?

Reply:
Sounds alright! It's probably not _correct_ because what about all the pickups where people didn't give feedback to, but hey.. maybe they just should do it.

'At least 1 kg was saved here' - not sure if this is ever used?